test-configs.yaml

```diff
@@ -179,6 +179,12 @@ test_plans:
     filters:
       - blocklist: *kselftest_defconfig_filter
 
+  preempt-rt:
+    rootfs: debian_buster_nfs
+    pattern: 'preempt-rt/{category}-{method}-{protocol}-nfs-preempt-rt-template.jinja2'
```

Comment:
You can use `root_type: nfs` instead of defining the pattern again.  See how it's done in #487.